Errors in typescript typings referencing nonexistent styles modules

Attempting to use the new modular typings from #7874 in `v1.0.0-beta.7`, I get a bunch of errors:

```
1 import { Breakpoint } from '../styles/breakpoints';
                             ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/material-ui/Dialog/withResponsiveFullScreen.d.ts(1,28): error TS2307: Cannot find module '../styles/breakpoints'.


5 import { Theme } from '../styles/theme';
                        ~~~~~~~~~~~~~~~~~

node_modules/material-ui/Drawer/Drawer.d.ts(5,23): error TS2307: Cannot find module '../styles/theme'.


4 import { Breakpoint } from '../styles/breakpoints';
                             ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/material-ui/Grid/Grid.d.ts(4,28): error TS2307: Cannot find module '../styles/breakpoints'.


3 import { Breakpoint } from '../styles/breakpoints';
                             ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/material-ui/Hidden/Hidden.d.ts(3,28): error TS2307: Cannot find module '../styles/breakpoints'.


3 import { Breakpoint } from '../styles/breakpoints';
                             ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/material-ui/Hidden/HiddenJs.d.ts(3,28): error TS2307: Cannot find module '../styles/breakpoints'.


3 import { Style, TextStyle } from '../styles/typography';
                                   ~~~~~~~~~~~~~~~~~~~~~~

node_modules/material-ui/Typography/Typography.d.ts(3,34): error TS2307: Cannot find module '../styles/typography'.


2 import { Theme } from './theme';
                        ~~~~~~~~~

node_modules/material-ui/styles/MuiThemeProvider.d.ts(2,23): error TS2307: Cannot find module './theme'.


1 import { Breakpoints } from './breakpoints';
                              ~~~~~~~~~~~~~~~

node_modules/material-ui/styles/createMuiTheme.d.ts(1,29): error TS2307: Cannot find module './breakpoints'.


2 import { Mixins } from './mixins';
                         ~~~~~~~~~~

node_modules/material-ui/styles/createMuiTheme.d.ts(2,24): error TS2307: Cannot find module './mixins'.


3 import { Palette } from './palette';
                          ~~~~~~~~~~~

node_modules/material-ui/styles/createMuiTheme.d.ts(3,25): error TS2307: Cannot find module './palette'.


7 import { Typography } from './typography';
                             ~~~~~~~~~~~~~~

node_modules/material-ui/styles/createMuiTheme.d.ts(7,28): error TS2307: Cannot find module './typography'.


2 export { default as createBreakpoints } from './breakpoints';
                                               ~~~~~~~~~~~~~~~

node_modules/material-ui/styles/index.d.ts(2,46): error TS2307: Cannot find module './breakpoints'.


3 export { default as createMuiTheme } from './theme';
                                            ~~~~~~~~~

node_modules/material-ui/styles/index.d.ts(3,43): error TS2307: Cannot find module './theme'.


4 export { default as createPalette } from './palette';
                                           ~~~~~~~~~~~

node_modules/material-ui/styles/index.d.ts(4,42): error TS2307: Cannot find module './palette'.


5 export { default as createTypography } from './typography';
                                              ~~~~~~~~~~~~~~

node_modules/material-ui/styles/index.d.ts(5,45): error TS2307: Cannot find module './typography'.


3 import { Theme } from './theme';
                        ~~~~~~~~~

node_modules/material-ui/styles/withStyles.d.ts(3,23): error TS2307: Cannot find module './theme'.


1 import { Theme } from './theme';
                        ~~~~~~~~~

node_modules/material-ui/styles/withTheme.d.ts(1,23): error TS2307: Cannot find module './theme'.


3 import { Theme } from '../styles/theme';
                        ~~~~~~~~~~~~~~~~~

node_modules/material-ui/transitions/Slide.d.ts(3,23): error TS2307: Cannot find module '../styles/theme'.


1 import { Breakpoint } from '../styles/breakpoints';
                             ~~~~~~~~~~~~~~~~~~~~~~~

node_modules/material-ui/utils/withWidth.d.ts(1,28): error TS2307: Cannot find module '../styles/breakpoints'.
```

--- a/docs/src/pages/customization/themes.md
+++ b/docs/src/pages/customization/themes.md
@@ -110,8 +110,7 @@ This component should preferably be used at **the root of your component tree**.
 ```jsx
 import React from 'react';
 import { render } from 'react-dom';
-import createMuiTheme from 'material-ui/styles/theme';
-import { MuiThemeProvider } from 'material-ui/styles';
+import { MuiThemeProvider, createMuiTheme } from 'material-ui/styles';
 import Root from './Root';
 
 const theme = createMuiTheme();

--- a/src/Dialog/withResponsiveFullScreen.d.ts
+++ b/src/Dialog/withResponsiveFullScreen.d.ts
@@ -1,4 +1,4 @@
-import { Breakpoint } from '../styles/breakpoints';
+import { Breakpoint } from '../styles/createBreakpoints';
 import { WithWidthProps } from '../utils/withWidth';
 
 export interface WithResponsiveFullScreenOptions {

--- a/src/Drawer/Drawer.d.ts
+++ b/src/Drawer/Drawer.d.ts
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { StyledComponent } from '..';
 import { ModalProps } from '../internal/Modal';
 import { SlideProps } from '../transitions/Slide';
-import { Theme } from '../styles/theme';
+import { Theme } from '../styles/createMuiTheme';
 
 export interface DrawerProps extends ModalProps {
   anchor?: 'left' | 'top' | 'right' | 'bottom';

--- a/src/Grid/Grid.d.ts
+++ b/src/Grid/Grid.d.ts
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
 import { HiddenProps } from '../Hidden/Hidden';
-import { Breakpoint } from '../styles/breakpoints';
+import { Breakpoint } from '../styles/createBreakpoints';
 
 export type GridAlignment = 'flex-start' | 'center' | 'flex-end' | 'stretch';
 

--- a/src/Hidden/Hidden.d.ts
+++ b/src/Hidden/Hidden.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
-import { Breakpoint } from '../styles/breakpoints';
+import { Breakpoint } from '../styles/createBreakpoints';
 
 export interface HiddenProps {
   only?: Breakpoint | Array<Breakpoint>;

--- a/src/Hidden/HiddenJs.d.ts
+++ b/src/Hidden/HiddenJs.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
-import { Breakpoint } from '../styles/breakpoints';
+import { Breakpoint } from '../styles/createBreakpoints';
 
 export interface HiddenJsProps {
   only?: Breakpoint | Array<Breakpoint>;

--- a/src/Typography/Typography.d.ts
+++ b/src/Typography/Typography.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent, PropTypes } from '..';
-import { Style, TextStyle } from '../styles/typography';
+import { Style, TextStyle } from '../styles/createTypography';
 
 export interface TypographyProps extends React.HTMLAttributes<HTMLElement> {
   align?: PropTypes.Alignment;

--- a/src/transitions/Collapse.d.ts
+++ b/src/transitions/Collapse.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
-import { Theme } from '../styles/theme';
+import { Theme } from '../styles/createMuiTheme';
 import { TransitionProps } from '../internal/Transition';
 
 export interface CollapseProps extends TransitionProps {

--- a/src/transitions/Fade.d.ts
+++ b/src/transitions/Fade.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
-import { Theme } from '../styles/theme';
+import { Theme } from '../styles/createMuiTheme';
 import { TransitionProps } from '../internal/Transition';
 
 export interface FadeProps extends TransitionProps {

--- a/src/transitions/Grow.d.ts
+++ b/src/transitions/Grow.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
-import { Theme } from '../styles/theme';
+import { Theme } from '../styles/createMuiTheme';
 import { TransitionProps } from '../internal/Transition';
 
 export interface GrowProps extends TransitionProps {

--- a/src/transitions/Slide.d.ts
+++ b/src/transitions/Slide.d.ts
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyledComponent } from '..';
-import { Theme } from '../styles/theme';
+import { Theme } from '../styles/createMuiTheme';
 import { TransitionProps } from '../internal/Transition';
 
 export interface SlideProps extends TransitionProps {

--- a/src/utils/withWidth.d.ts
+++ b/src/utils/withWidth.d.ts
@@ -1,4 +1,4 @@
-import { Breakpoint } from '../styles/breakpoints';
+import { Breakpoint } from '../styles/createBreakpoints';
 export interface WithWidthOptions {
   resizeInterval: number;
 }

